src/CHANGES.txt

```diff
@@ -162,6 +162,10 @@ RELEASE 3.1.0.alpha.yyyymmdd - NEW DATE WILL BE INSERTED HERE
       Three uses of variables not defined are changed.
     - Some script changes in trying to find scons engine
 
+  From Bernhard M. Wiedemann:
+    - Allow to override build date with SOURCE_DATE_EPOCH
```

Comment:
Please add a note that this is only about building SCons packages and not universal to builds done by scons when building other software.